client/planning-extension/src/extension.ts

```diff
@@ -269,17 +271,24 @@ const extension: IExtension = {
             },
         };
 
+        const allVocabularies = superdesk.entities.vocabulary.getAll();
+
+        if (allVocabularies.has(PLACE_FIELD_DEPENDENT_VOCABULARY_ID) === false) {
+            result.contributions.getInstanceConfigurationIssues = () => (
```

Comment:
another approach is to define `getInstanceConfigurationIssues` unconditionally in `result.contributions` and return an empty array if there are no issues.